.github/workflows/build-napi.yml

```diff
@@ -84,6 +84,10 @@ jobs:
         with:
           key: ${{ matrix.settings.target }}-cargo-${{ matrix.settings.os }}
 
+      - name: Generate schemas
+        run: npm ci && npm run schemas
+        working-directory: .
+
```

Comment:
We re-run `npm ci` twice. We can put it below the next step instead.

Reply:
They are being run at different directories though:
The `Generate schemas` step is being run at the root of the repository, where `quicktype` code generation dependency is.
The `Install dependencies` step is being run inside the `crates/bitwarden-napi` folder, that's where the NAPI dependencies are.

Reply:
Shouldn't schema generation be a separate workflow with a published artifact? We're going to want these schemas in more than just the napi builds

Reply:
Separated into a new workflow that publishes the schema artifacts, let me know what you think.